Fruitties/androidApp/src/main/java/com/example/fruitties/android/ui/ListScreen.kt

```diff
@@ -108,7 +108,7 @@ fun ListScreen() {
             var expanded by remember { mutableStateOf(false) }
             Row(modifier = Modifier.padding(16.dp)) {
                 Text(
-                    text = "Cart has ${cartState.itemList.count()} items",
+                    text = "Item types in cart: ${cartState.cartDetails.count()}",
```

Comment:
Why did we change the text?

Reply:
I you have 10 apples in your cart, it said "Cart has 1 items." When I added items and the number did not increase, I thought I had a bug, but I was just by the English sentence.

Reply:
Actually, I think it will be better to add up the total count of all items. I'll update the UI on Android and iOS.